src/ClassicUO.Assets/TileDataLoader.cs

```diff
@@ -447,12 +451,11 @@ public struct LandGroupNew
     }
 
     [StructLayout(LayoutKind.Sequential, Pack = 1)]
-    public struct LandTilesNew
+    public unsafe struct LandTilesNew
     {
-        public TileFlag Flags;
+        public UInt64 Flags;
```

Comment:
Should we declare `TileFlag` is a `ulong` enum instead? Also I tend to use primitives like `ulong` instead of `UInt64` (the struct backing the primitive).

Reply:
I thought UInt64 explicitly states the size, but apparently C# (unlike C and C++) hard-codes the size of all basic integer types already.
Which means by your wish, all uses of `IntPtr` should be changed to `nint`.

Reply:
> Should we declare `TileFlag` is a `ulong` enum instead?

No, because the constructor takes an `ulong` (for whatever reason), so we'd need to cast the enum to an ulong and then back to the enum. That doesn't make sense. We could change the constructor to that enum which would make sense, but then we still given that there are two different sizes, we'd need two different enum declarations.

Reply:
yeah, it only makes sense if the enum is ulong also. It's fine to keep it the way it is.